client/hotrod-client/src/main/java/org/infinispan/client/hotrod/ServerStatistics.java

```diff
@@ -18,17 +18,35 @@
 
    /**
     * Number of entries currently in the Hot Rod server
+    * @deprecated Since 14.0, please use {@link #APPROXIMATE_ENTRIES}
     */
+   @Deprecated
    String CURRENT_NR_OF_ENTRIES = "currentNumberOfEntries";
 
    /**
     * Number of entries stored in Hot Rod server
     * since the server started running.
+    * @deprecated Since 13.0, please use {@link #STORES} instead.
     */
+   @Deprecated
    String TOTAL_NR_OF_ENTRIES = "totalNumberOfEntries";
 
    /**
-    * Number of put operations.
+    * Approximate number of entry copies currently in the server cache.
```

Comment:
I'm not sure what this means :fearful: 
is it the entries on a single server? or cluster-wide? does it include persistency? 

why `Entries owned by multiple nodes are counted once for each owner`? Wouldn't be better to sum everything a divide by the `num-owners`?

ps. I haven't checked impl yet.

Reply:
ps2. why do we need 2 `APPROXIMATE_ENTRIES` and `APPROXIMATE_ENTRIES_UNIQUE`?

Reply:
> is it the entries on a single server? or cluster-wide? 

I kept the old javadoc style, which is modeled on `Stats`.
Because `Stats` is supposed to be used as an interface for both a single server and the cluster, the javadocs don't say explicitly that they apply to either one.

`Encoder2x.statsResponse()` does write cluster-wide statistics with different name (e.g. `approximateEntries` becomes `globalApproximateEntries`), but since they didn't have constants in `ServerStatistics` before I didn't want to add them now. Perhaps I should...

> does it include persistency?

Yes, I'll add that info.

> why Entries owned by multiple nodes are counted once for each owner? Wouldn't be better to sum everything a divide by the num-owners?
> ps2. why do we need 2 APPROXIMATE_ENTRIES and APPROXIMATE_ENTRIES_UNIQUE?

That's the difference between APPROXIMATE_ENTRIES and APPROXIMATE_ENTRIES_UNIQUE: the "unique" stat only counts entries owned as primary owner, so there are no duplicates.

The idea was to let the user know when there's a state transfer in progress and the cache is holding more entry copies than expected.